### DISCUSSION TITLE
Health insurance can be cheesed by self-suffocating

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13698] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Health insurance can simply be activated by equipping an empty diving suit and waiting 10 seconds then taking it off every round for free money. No kink shaming but I think it's a little weird

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/794d5534-0d1b-4382-9c00-9def85d593fb)

Suggest changing trigger condition

From "when your health drops below 30%"
To "if no character goes unconscious" and potentially increasing the reward since it's harder

something like 

    <AbilityGroupInterval interval="1">
      <Conditions>
        <AbilityConditionCrewMemberUnconscious />
      </Conditions>
      <Abilities>
        <CharacterAbilityGivePermanentStat statidentifier="healthinsurance" value="1" setvalue="true" />
      </Abilities>
    </AbilityGroupInterval>
    <AbilityGroupEffect abilityeffecttype="OnRoundEnd">
      <Conditions>
        <AbilityConditionHasPermanentStat statidentifier="healthinsurance" min="1" invert="true"/>
      </Conditions>
      <Abilities>
        <CharacterAbilityGiveMoney amount="200"/>
        <CharacterAbilityResetPermanentStat statidentifier="healthinsurance" />
      </Abilities>
    </AbilityGroupEffect> 

### Reproduction steps

givetalent "Health Insurance"
equip suit
remove o2
wait for <30% health
remove suit
free money

### Version

v1.4.0.0 (unstable)

**Comment by HawkinsThe71st** [comment]
Also the effect triggers after rounds without missions since `abilityeffecttype="OnRoundEnd"` is used instead of `abilityeffecttype="OnAnyMissionCompleted"`

**Comment by Vykn** [comment]
Honestly the talent sucks as it is - nobody is going to pick it. 100 marks each round is pitiful compared to the thousands you get from missions. 
Good if you want to pad the talent list maybe, but does not give the players the sense that their choices are meaningful when there are clearly worse options than others.

**Reply by JeroonBosch** [reply]
It also discounts all medicines by 10%, which will end up being a much bigger benefit than 100 Mk and imo is enough power for a Tier 1 talent.

**Reply by Vykn** [reply]
Ah, I completely forgot about that aspect of the talent. That's pretty fair then

**Reply by ubertpendragon** [reply]
My point was

`Talent you can trigger 100% of the time for a 100mk` **<<<** `Talent you can trigger 33% of the time for 300mk`

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5115/commits/6590a97da32301fc7eb762be68ee87b3277375d2 (commit in our private repo)
